README.rst

```diff
@@ -247,6 +247,47 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_CONFIG_MODIFIED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+Whether ``config_modified`` checks are created automatically for devices.
+
+``OPENWISP_MONITORING_CONFIG_MODIFIED_MAX_TIME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    |  ``int``  |
++--------------+-----------+
+| **default**: |   ``5``   |
++--------------+-----------+
+
+After ``config`` is ``modified``, if the ``modified`` status does not change  after a
```

Comment:
Sounds very technical too.

What about something like

> This setting allows configuring the time that the check is allowed to fail until the device health status changes to ``problem``

It is important to take the necessary time to re-read the documentation multiple times to make sure it is written properly and others can understand it. It is not easy though!